configuration/exodus-pipeline.yaml

```diff
@@ -450,123 +462,7 @@ Resources:
-        - PolicyName: !Sub ${project}-codebuild-policy-${env}
-          PolicyDocument:
-            Version: 2012-10-17
-            Statement:
-              - Effect: Allow
-                Action:
-                  - s3:*
-                  - codebuild:*
-                  - logs:CreateLogGroup
-                  - logs:CreateLogStream
-                  - logs:PutLogEvents
-                Resource: "*"
-      Path: /
-
-  CloudFormationRole:
-    Type: AWS::IAM::Role
-    Properties:
-      RoleName: !Sub ${project}-cloudform-${env}
-      AssumeRolePolicyDocument:
-        Version: 2012-10-17
-        Statement:
-          - Effect: Allow
-            Action:
-              - sts:AssumeRole
-            Principal:
-              Service:
-                - cloudformation.amazonaws.com
-      Policies:
-        - PolicyName: !Sub ${project}-cloudform-policy-${env}
-          PolicyDocument:
-            Version: 2012-10-17
-            Statement:
-              - Effect: Allow
-                Action:
-                  - iam:*
-                  - lambda:*
-                  - s3:*
-                  - cloudfront:*
-                  - cloudwatch:*
-                  - cloudformation:*
-                Resource:
-                  - "*"
-      Path: /
-
-  CodePipelineRole:
-    Type: AWS::IAM::Role
-    Properties:
-      RoleName: !Sub ${project}-codepipe-${env}
-      AssumeRolePolicyDocument:
-        Version: 2012-10-17
-        Statement:
-          - Effect: Allow
-            Action:
-              - sts:AssumeRole
-            Principal:
-              Service:
-                - codepipeline.amazonaws.com
-      Policies:
-        - PolicyName: !Sub ${project}-codepipe-policy-${env}
-          PolicyDocument:
-            Version: 2012-10-17
-            Statement:
-              - Effect: Allow
-                Action:
-                  - iam:PassRole
-                  - sns:*
-                  - lambda:ListFunctions
-                  - lambda:InvokeFunction
-                  - s3:*
-                  - cloudwatch:*
-                  - cloudformation:*
-                  - codebuild:StartBuild
-                  - codebuild:BatchGetBuilds
-                Resource: "*"
-      Path: /
 
-  CloudWatchEventRole:
-    Type: AWS::IAM::Role
-    Condition: IsProd
-    Properties:
-      RoleName: !Sub ${project}-cloudwatch-${env}
-      AssumeRolePolicyDocument:
-        Version: 2012-10-17
-        Statement:
-          - Effect: Allow
-            Action:
-              - sts:AssumeRole
-            Principal:
-              Service:
-                - events.amazonaws.com
-      Policies:
-        - PolicyName: !Sub ${project}-cloudwatch-policy-${env}
-          PolicyDocument:
-            Version: 2012-10-17
-            Statement:
-              - Effect: Allow
-                Action: codepipeline:StartPipelineExecution
-                Resource: !Sub arn:aws:codepipeline:${region}:${AWS::AccountId}:${CodePipeline}
```

Comment:
+1 for the idea of capturing the roles in a new .yaml file for now.  I also agree it doesn't have to be included in scope of this PR, but it would be valuable.